katcp/core.py

```diff
@@ -752,9 +758,9 @@ def __init__(mcs, name, bases, dct):
         Parameters
         ----------
         mcs : class
-            The metaclass instance
+            The meta-class instance
```

Comment:
What's the motivation for changing the spelling? Python spells it "metaclass", both in documentation and in the `metaclass=` syntax.

Reply:
It was due to my IDE auto-spellcheck fix thingy.
Will fix it.